MANIFEST.in

```diff
@@ -1,15 +0,0 @@
-include *.yml
-include LICENSE
-include MANIFEST.in
-include pyproject.toml
-include tox.ini
-
-recursive-include .circleci *
-
-recursive-include docs *
-prune docs/_build
-
-recursive-include references *.pdf
-
-recursive-include tests *.py
-recursive-include tests/data *
```

Comment:
Don't think including tests in the wheel makes sense personally, but maybe there's a good reason to that I don't know about. Left them out for now.